### PR TITLE
Flow readme section

### PR DESCRIPTION
**Pull request by amandalindsay**
# Flow doc updates

This addresses the following tickets: HZX-349, 347, and 337 to add:
- Orbital's new Readme.md section
- Document the Flow UI and the Designer feature
- Update the 'Enable role-based authorisation' info

New rebranded graphics have been added.

### CODE REVIEW

File: docs/modules/guides/pages/work-with-xml.adoc [hunk 1]
```diff
@@ -85,10 +85,12 @@ With {short-product-name}, instead of using integration code, schemas are used t
 
 First, write Taxi code that describes the Films Service. This is an HTTP operation that returns a list of films.
 
+TIP: Use {short-product-name}'s *Designer* to create and edit your Taxi code. On the left-hand side, input a data sample or upload a CSV, TSV, PSV, JSON or XML source file. On the right-hand side, create a Taxi schema based on this data. The parsed results appear below as you work. When you are happy with your Taxi model, you can copy it from the Designer into your project.
```
Comment (Rob-Hazelcast) [inline docs/modules/guides/pages/work-with-xml.adoc:88]: Is it obvious how to access Designer?

File: docs/modules/introduction/pages/about-the-UI.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+= A look at the UI
+:description: An overview of the user interface
+
+This topic provides an overview of Hazelcast {short-product-name}'s user interface. 
+
+image:Flow-welcome-screen.png[]
+
+The {short-product-name} UI provides the following main screens and functions:
```
Comment (Rob-Hazelcast) [inline docs/modules/introduction/pages/about-the-UI.adoc:8]: These bullets should all end in full stops.

IMO, the full stops after the bolded terms looks odd. I'd use an em dash or a colon, or maybe just a space.

File: docs/modules/workspace/pages/overview.adoc [hunk 3]
```diff
@@ -230,6 +230,72 @@ git {
 }
 ----
 
+== Include a README.md file
+
+If you supply a README.md file at the root of the project, it will be read by {short-product-name} and displayed in the UI.
```
Comment (Rob-Hazelcast) [inline docs/modules/workspace/pages/overview.adoc:235]: Nice and concise, this could easily have been several sentences

**Review by Rob-Hazelcast — APPROVED** [review]
Looks great. Couple of minor comments.

**Review by pollett — APPROVED** [review]
Great additions, all looks good, thanks Amanda.

Should be all fine to hit merge on 👍

**Pull request merged**